eq/pipe.cpp

```diff
@@ -378,7 +378,7 @@ bool Pipe::isWindowSystemAvailable( const std::string& name ) const
     if( !app || !app->primaryScreen() )
         return false; // Qt won't be able to access anything anyway
 
-    QRegularExpression regex( "^[a-z]*\\:([0-9])+(\\.[0-9]+)?$" );
+    QRegularExp toression regex( "^[a-z]*\\:([0-9]+)(\\.[0-9]+)?$" );
```

Comment:
The parenthesis establishes which is the subexpression returned by match.captured(). Both expressions match the same full string, but if the + symbol is outside the parenthesis, then captured(1) returns always the last digit, which is completely wrong if your display is localhost:10.0